### PR TITLE
Example pull request

### PR DESCRIPTION
**Pull request by priorax**

### CODE REVIEW

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
```
Comment (priorax) [inline .github/workflows/nodejs.yml:1]: If you had issues enabled, I could say that this PR is related to an issue, instead I'll link it back to a PR #1

File: .github/workflows/nodejs.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
```
Comment (neilospricey) [inline .github/workflows/nodejs.yml:14]: Blah blah

File: src/filterEmoji.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import emojiList from "./emojiList.json";
 export default function filterEmoji(searchText, maxResults) {
   return emojiList
     .filter(emoji => {
-      if (emoji.title.toLowerCase().includes(searchText.toLowerCase())) {
+      if (emoji.title.toLocaleLowerCase().includes(searchText.toLocaleLowerCase())) {
         return true;
```
Comment (neilospricey) [inline src/filterEmoji.js:7]: Friendly code review :-)

File: .github/workflows/nodejs.yml [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
```
Comment (priorax) [inline .github/workflows/nodejs.yml:4]: Also, unlike our current tools, 

# MARKDOWN WORKS!

This means you can do things like [link to other parts of the code](https://github.com/neilospricey/emoji-search/blob/master/Dockerfile) quite easily

**Comment by neilospricey** [comment]
Closing without merging

**Pull request closed without merging**